operator/data/examples/multicluster/values-istio-multicluster-primary.yaml

```diff
@@ -2,31 +2,33 @@ apiVersion: install.istio.io/v1alpha1
 kind: IstioOperator
 spec:
   values:
-    gateways:
-      istio-ingressgateway:
-        env:
-          ISTIO_META_NETWORK: "network1"
```

Comment:
Do we need this environment variable on the ingressgateway?

Reply:
not in any of our docs that have been validated.  this config is already in mesh config.